package.json

```diff
@@ -171,7 +171,8 @@
     "save": "^2.4.0",
     "semver": "^7.5.2",
     "shim-keyboard-event-key": "^1.0.3",
-    "underscore": "^1.13.1"
+    "underscore": "^1.13.1",
+    "expo-av": "~13.4.1"
```

Comment:
Not sure where to put this but I'm seeing this in the `npm run web` output, it seems to be related given that expo-av shows up in the stacktrace.

```
[web-server] ERROR in ./node_modules/expo-modules-core/build/uuid/uuid.web.js 7:10-27
[web-server] Module not found: Error: Can't resolve 'crypto' in '~/Expensidev/App/node_modules/expo-modules-core/build/uuid'
[web-server]
[web-server] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[web-server] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[web-server]
[web-server] If you want to include a polyfill, you need to:
[web-server] 	- add a fallback 'resolve.fallback: { "crypto": require.resolve("crypto-browserify") }'
[web-server] 	- install 'crypto-browserify'
[web-server] If you don't want to include a polyfill, you can use an empty module like this:
[web-server] 	resolve.fallback: { "crypto": false }
[web-server] resolve 'crypto' in '~/Expensidev/App/node_modules/expo-modules-core/build/uuid'
[web-server]   Parsed request is a module
[web-server]   using description file: ~/Expensidev/App/node_modules/expo-modules-core/package.json (relative path: ./build/uuid)
[web-server]     Field 'browser' doesn't contain a valid alias configuration
[web-server]     resolve as module
[web-server]       ~/Expensidev/App/node_modules/expo-modules-core/build/uuid/node_modules doesn't exist or is not a directory
[web-server]       ~/Expensidev/App/node_modules/expo-modules-core/build/node_modules doesn't exist or is not a directory
[web-server]       ~/Expensidev/App/node_modules/expo-modules-core/node_modules doesn't exist or is not a directory
[web-server]       ~/Expensidev/App/node_modules/node_modules doesn't exist or is not a directory
[web-server]       looking for modules in ~/Expensidev/App/node_modules
[web-server]         single file module
[web-server]           using description file: ~/Expensidev/App/package.json (relative path: ./node_modules/crypto)
[web-server]             no extension
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto doesn't exist
[web-server]             .web.js
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto.web.js doesn't exist
[web-server]             .website.js
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto.website.js doesn't exist
[web-server]             .js
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto.js doesn't exist
[web-server]             .jsx
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto.jsx doesn't exist
[web-server]             .web.ts
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto.web.ts doesn't exist
[web-server]             .website.ts
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto.website.ts doesn't exist
[web-server]             .website.tsx
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto.website.tsx doesn't exist
[web-server]             .ts
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto.ts doesn't exist
[web-server]             .web.tsx
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto.web.tsx doesn't exist
[web-server]             .tsx
[web-server]               Field 'browser' doesn't contain a valid alias configuration
[web-server]               ~/Expensidev/App/node_modules/crypto.tsx doesn't exist
[web-server]         ~/Expensidev/App/node_modules/crypto doesn't exist
[web-server]       ~/Expensidev/node_modules doesn't exist or is not a directory
[web-server]       ~/node_modules doesn't exist or is not a directory
[web-server]       /Users/node_modules doesn't exist or is not a directory
[web-server]       /node_modules doesn't exist or is not a directory
[web-server]  @ ./node_modules/expo-modules-core/build/uuid/index.js 1:0-33 1:0-33
[web-server]  @ ./node_modules/expo-modules-core/build/index.js 9:0-41 9:0-41
[web-server]  @ ./node_modules/expo-av/build/ExponentAV.web.js 1:0-79 10:23-47 12:23-46 14:23-52 19:15-44 157:12-41 163:12-41 266:17-41 268:28-52 273:17-40 275:28-51 289:24-48 297:24-47
[web-server]  @ ./node_modules/expo-av/build/Audio.js 2:0-38 52:17-40
[web-server]  @ ./node_modules/expo-av/build/index.js 1:0-33 2:0-17
[web-server]  @ ./src/components/VideoPlayer/propTypes.js 1:0-37 19:14-32
[web-server]  @ ./src/components/VideoPlayer/index.js 4:0-76 13:24-44 14:27-50
[web-server]  @ ./src/components/VideoPlayerPreview/index.js 6:0-50 77:95-106
[web-server]  @ ./src/components/HTMLEngineProvider/HTMLRenderers/VideoRenderer.js 5:0-64 25:42-60
[web-server]  @ ./src/components/HTMLEngineProvider/HTMLRenderers/index.js 9:0-44 19:9-22
 @ ./src/components/HTMLEngineProvider/BaseHTMLEngineProvider.tsx 10:0-44 119:15-28
[web-server]  @ ./src/components/HTMLEngineProvider/index.tsx 4:0-62 9:42-64
[web-server]  @ ./src/App.js 14:0-65 55:162-180
[web-server]  @ ./index.js 5:0-28 9:9-12
```

Reply:
I've missed that we have updated Expo SDK version on main and Expo-AV didn't match that version. Now I'm going through the package.json and checking if everything has the correct version.